### ISSUE TITLE
SRW - revise options/calculation for Flux and Intensity reports

### ISSUE THREAD

**Issue by mrakitin** [radiasoft/sirepo#189]
Calculation of Flux and Intensity in Sirepo should reproduce the following SRW calculations with Python scripts with the following options:

Spectral Flux Report 
--
In SR Calculator:
```bash
--sm --und_by=0.8 --und_bx=0 --sm_mag=1 --sm_meth=-1 --sm_hi=1 --sm_hf=21 --sm_prl=1 --sm_pra=1
```
In Expert Mode:
- Fast, but Approximate UR Calculation method:
```bash
--sm --und_b=0.8 --sm_mag=1 --sm_meth=-1 --sm_hi=1 --sm_hf=21 --sm_prl=1 --sm_pra=1
--sm --und_g=7.0 --sm_mag=1 --sm_meth=-1 --sm_hi=1 --sm_hf=21 --sm_prl=1 --sm_pra=1 (uses mag. field files)
```

- Slow, but Accurate SR Calculation method:
```bash
--sm --und_b=0.8 --sm_mag=1 --sm_meth=1 --sm_prec=0.02
--sm --und_g=7.0 --sm_mag=2 --sm_meth=1 --sm_prec=0.02 (uses mag. field files)
```
In all other cases raise an exception (for the moment).

Single-E Spectral Flux per Unit Surface vs E Report
--

In SR Calculator:
```bash
--ss --und_b=0.8 --ss_mag=1 --ss_meth=1 --ss_prec=0.02
```
In Expert Mode:
```bash
--ss --und_b=0.8 --ss_mag=1 --ss_meth=1 --ss_prec=0.02
--ss --und_g=7.0 --ss_mag=2 --ss_meth=1 --ss_prec=0.02
```

Single-E/Multi-E Spectral Flux per Unit Surface vs X and Y (Intensity Distribution) Report
--

In SR Calculator:
```bash
--si --und_b=0.8 --w_mag=1 --w_meth=1 --w_prec=0.02 --si_type=0 --w_e=9122 --w_smpf=-1 --w_nx=100 --w_ny=100
--si --und_b=0.8 --w_mag=1 --w_meth=1 --w_prec=0.02 --si_type=1 --w_e=9122 --w_smpf=-1 --w_nx=100 --w_ny=100
```
In Expert Mode:
```bash
--si --und_b=0.8 --w_mag=1 --w_meth=1 --w_prec=0.02 --si_type=0 --w_e=9122 --w_smpf=-1 --w_nx=100 --w_ny=100
--si --und_b=0.8 --w_mag=1 --w_meth=1 --w_prec=0.02 --si_type=1 --w_e=9122 --w_smpf=-1 --w_nx=100 --w_ny=100

--si --und_g=7.0 --w_mag=2 --w_meth=1 --w_prec=0.02 --si_type=0 --w_e=8487 --w_smpf=-1 --w_nx=100 --w_ny=100 (uses mag. field files)
--si --und_g=7.0 --w_mag=2 --w_meth=1 --w_prec=0.02 --si_type=1 --w_e=8487 --w_smpf=-1 --w_nx=100 --w_ny=100
```

**Comment by mrakitin** [comment]
Attaching the results for Flux report in calculator mode - [flux_report_calc_mode.zip](https://github.com/radiasoft/sirepo/files/261648/flux_report_calc_mode.zip). The archive contains the following files/folders:

- `srw/`:
 - `srx.py` script, which generates `data_SRX/res_spec_me.dat` file, being submitted with the following options:
  ```bash
--sm --und_b=0.8 --sm_mag=1 --sm_meth=-1 --sm_hi=1 --sm_hf=21 --sm_prl=1 --sm_pra=1
```
- `sirepo/`:
 - `flux_report_calc_mode.json` JSON file for the corresponding calculation in Sirepo;
 - `exported.dat` - exported file from Sirepo, which is identical to `srw/data_SRX/res_spec_me.dat`;
- `flux_srw.png` - the resulted plot for the Flux report in SRW;
- `flux_sirepo.png` - the resulted plot for the Flux report in Sirepo.

**Comment by mrakitin** [comment]
Fixed/implemented in 81654b8, 91ac958, 83d69b6 and 061ea1d. If there is an issue, a separate ticket should be created. Closing.
